src/pageComponents/HomePage/blocks/Hero/Hero.tsx

```diff
@@ -1,14 +1,141 @@
+	const [sliderRef, instanceRef] = useKeenSlider<HTMLDivElement>({
+		initial: 0,
+		slideChanged(slider) {
+			setCurrentSlide(slider.track.details.rel);
+		},
+		created() {
+			setLoaded(true);
+		},
+	});
+	const { isScreenTabletSm } = useScreenQuery();
+	const buttons = (
+		<div className={s.buttonsGap}>
+			{/* <ButtonLink href="#GetHelp">Отримати допомогу</ButtonLink> */}
+			<Button type="secondary">Допомогти нам</Button>
+		</div>
+	);
+	const dots = loaded && instanceRef.current && (
+		<div className={s.dots}>
+			{[...Array(instanceRef.current.track.details.slides.length).keys()].map((idx) => {
+				return (
+					<button
+						key={idx}
+						onClick={() => {
+							instanceRef.current?.moveToIdx(idx);
+						}}
+						className={clsx(s.dot, currentSlide === idx ? `${s.active}` : '')}
+					></button>
+				);
+			})}
+		</div>
+	);
+
+	const arrows = (
+		<div className={s.arrows}>
+			{loaded && instanceRef.current && (
+				<div className={s.arrowsGap}>
+					<Arrow
+						left
+						onClick={(e: any) => e.stopPropagation() || instanceRef.current?.prev()}
+						disabled={currentSlide === 0}
+					/>
+
+					<Arrow
+						onClick={(e: any) => e.stopPropagation() || instanceRef.current?.next()}
+						disabled={currentSlide === instanceRef.current.track.details.slides.length - 1}
+					/>
+				</div>
+			)}
+		</div>
+	);
+	console.log(isScreenTabletSm);
+	function renderNavigation() {
+		return isScreenTabletSm ? arrows : dots;
+	}
 	return (
-		<Section className={s.Hero}>
-			<Container>
-				<h2>Hero</h2>
-			</Container>
-		</Section>
+		<Container className={s.positionRelative}>
+			<div ref={sliderRef} className="keen-slider">
+				<div className="keen-slider__slide">
+					<div className={clsx(s.content, s.banner1)}>
+						<div className={s.text}>
+							<Text variant="h1" className={clsx(s.heading, s.blueColor)}>
+								Надаємо гуманітарні набори потребуючим
+							</Text>
+							<Text variant="various3">м. Кривий Ріг</Text>
+						</div>
+						{buttons}
+					</div>
+				</div>
+				<div className="keen-slider__slide">
+					<div className={clsx(s.content, s.banner2)}>
+						<div className={s.text}>
+							<Text variant="h1" className={clsx(s.whiteColor, s.headingWidth)}>
+								Відбудовуємо зруйновані та пошкоджені об’єкти (Херсонська область)
+							</Text>
+							<Text variant="various3" className={s.whiteColor}>
+								м. Кривий Ріг
+							</Text>
+						</div>
+						{buttons}
+					</div>
+				</div>
+				<div className="keen-slider__slide">
+					<div className={clsx(s.content, s.banner3)}>
+						<div className={s.text}>
+							<Text variant="h1" className={clsx(s.whiteColor, s.headingWidth)}>
+								Забезпечуємо медичні заклади м. Кривий Ріг, Криворізького району та Херсонської області
+							</Text>
+							<Text variant="various3" className={s.whiteColor}>
+								м. Кривий Ріг
+							</Text>
+						</div>
+						{buttons}
+					</div>
+				</div>
+			</div>
```

Comment:
I think better create const with data of sliders. After that Array.map()

Reply:
I don't think it's appropriate - because I can run through the array, but at each iteration, different styles must be added to the heading and city
it will only turn out worse with an array in my opinion

Reply:
I did it in some different way, please check it in next commit